templates/home/api.html

```diff
@@ -21,121 +21,147 @@
-	&lt;div class="container"&gt;
-		&lt;div class="innerbox"&gt;
-			&lt;h1 property="name" id="wb-cont"&gt;Canada.ca&lt;/h1&gt;
-			&lt;p&gt;The official website of the Government of Canada&lt;/p&gt;
+	<summary>Version 2.0 - Promotional banner</summary>
+	<ul>
+		<li>The background image is loaded with the provisional plugin bgimg which set the background image. The background image is hosted in the WMS at this location. https://www.canada.ca/content/dam/canada/carousel/bkg-home-banner.jpg.</li>
+		<li>The intend is to publish background image that represent natural Canadian landscape and may get updated montly or seasonal.</li>
+	</ul>
+	<pre><code>&lt;div class="provisional bg-cover" data-bgimg="https://www.canada.ca/content/dam/canada/carousel/bkg-home-banner.jpg"&gt;
+		&lt;div class="container p-0 p-sm-3"&gt;
+			&lt;div class="well header-rwd brdr-0 brdr-rds-0 text-white bg-gctheme"&gt;
+				&lt;h1 property="name" id="wb-cont"&gt;Canada.ca&lt;/h1&gt;
+				&lt;p class="mrgn-tp-md"&gt;The official website of the Government of Canada&lt;/p&gt;
+			&lt;/div&gt;
 		&lt;/div&gt;
-	&lt;/div&gt;
-	&lt;div class="promo"&gt;
-		&lt;div class="container"&gt;
-			&lt;a class="figcaption" href="#"&gt;
-				&lt;figure class="clearfix"&gt;
-					&lt;div&gt;&lt;p class="pull-left mrgn-tp-md mrgn-bttm-md"&gt;Featured:&lt;/p&gt;&lt;/div&gt;
-					&lt;figcaption class="pull-left mrgn-tp-md mrgn-bttm-md"&gt;[Promotion title]&lt;/figcaption&gt;
-				&lt;/figure&gt;
-			&lt;/a&gt;
+		&lt;div class="bg-gctheme opct-90"&gt;
+			&lt;div class="container"&gt;
+				&lt;a class="figcaption text-white" href="#"&gt;
+					&lt;figure class="clearfix"&gt;
+						&lt;div&gt;&lt;p class="pull-left mrgn-tp-md mrgn-bttm-md"&gt;Featured:&lt;/p&gt;&lt;/div&gt;
+						&lt;figcaption class="pull-left mrgn-tp-md mrgn-bttm-md"&gt;[Promotion title]&lt;/figcaption&gt;
+					&lt;/figure&gt;
+				&lt;/a&gt;
+			&lt;/div&gt;
 		&lt;/div&gt;
-	&lt;/div&gt;
-&lt;/div&gt;</code></pre>
-
-<h5>Provisional</h5>
-<p>Support for a set of predefined color to match the season. The intend is to set the color of the inner box with the site title to match the promotional banner background color.</p>
-
-<p>The promotional color theme are set on the <code>div.home-banner</code>. Here the supported theme with their corresponding CSS class name</p>
-<ul>
-	<li>(default) Sky - <code>sky</code></li>
-	<li>Earth - <code>earth</code></li>
-	<li>Vegetation - <code>vegetation</code></li>
-	<li>Night - <code>night</code></li>
-	<li>Forest - <code>forest</code></li>
-	<li>Autumn - <code>autumn</code></li>
-</ul>
-
+	&lt;/div&gt;</code></pre>
 </details>
 
-<h2>Evaluation and report</h2>
-<p>There is no report</p>
-
-<h2>Previous version</h2>
-<ul>
-	<li>See GCWeb released with WET 4.0.29. All code change are documented in the migration instruction of GCWeb new look - V5.</li>
-</ul>
 
-<h2>Test cases</h2>
-<ul>
-	<li>Removed: <del>Provisional - Test-1: Promotional color theme</del></li>
-</ul>
```

Comment:
Keep for historical purpose. I do think that test-1 file still exist

Reply:
Done. Thanks